client/components/mma/accountoverview/ProductCard.tsx

```diff
@@ -265,20 +265,22 @@ export const ProductCard = ({
 					)}
 				</Card.Header>
 
-				{cardConfig.showBenefitsSection && nextPaymentDetails && (
-					<Card.Section backgroundColor="#edf5fA">
-						<p css={benefitsTextCss}>
-							You’re supporting the Guardian with{' '}
-							{nextPaymentDetails.currentPriceValue} per{' '}
-							{nextPaymentDetails.paymentInterval}, and have
-							access to exclusive extras.
-						</p>
-						<BenefitsToggle
-							productType={specificProductType.productType}
-							subscriptionPlan={mainPlan}
-						/>
-					</Card.Section>
-				)}
+				{/* ---- */}
+				{(cardConfig.showBenefitsSection ||
+					cardConfig.showDigitalBenefitsSection) &&
+					nextPaymentDetails && (
+						<Card.Section backgroundColor="#edf5fA">
+							<p css={benefitsTextCss}>
+								{cardConfig.showDigitalBenefitsSection
```

Comment:
is there ever a situation where `showDigitalBenefitsSection` and `showBenefitsSection` could both be true?